cmd/kube-apiserver/app/server.go

```diff
@@ -299,7 +299,15 @@ func Run(s *options.ServerRunOptions) error {
 	}
 
 	admissionControlPluginNames := strings.Split(s.GenericServerRunOptions.AdmissionControl, ",")
-	pluginInitializer := kubeadmission.NewPluginInitializer(client, sharedInformers, apiAuthorizer)
+	var cloudConfig []byte
+
+	if s.CloudProvider.CloudConfigFile != "" {
+		cloudConfig, err = ioutil.ReadFile(s.CloudProvider.CloudConfigFile)
```

Comment:
Ack, just open up an issue as a reminder to do this clean up.

SInce this is a bug fix, post-code freeze merge is fine (marking it for 1.6).

Will leave review and signoff to @msau42